src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests

```diff
@@ -1,78 +0,0 @@
-# This file was generated by Autom4te Thu Apr 10 10:06:43 UTC 2014.
```

Comment:
@dev-gh this files changes redundant

Reply:
@LuxoftAKutsan 
These files were removed with last commits

Reply:
@dev-gh I still can see changes